modules/tinymce/src/themes/silver/main/ts/ui/window/SilverDialogInstanceApi.ts

```diff
@@ -85,12 +85,8 @@ const getDialogApi = <T extends Dialog.DialogData>(
     });
   };
 
-  const disable = (name: string) => {
-    getCompByName(access, name).each(Disabling.disable);
-  };
-
-  const enable = (name: string) => {
-    getCompByName(access, name).each(Disabling.enable);
+  const setEnabled = (name: string, state: boolean) => {
+    getCompByName(access, name).each(state === true ? Disabling.enable : Disabling.disable);
```

Comment:
```suggestion
    getCompByName(access, name).each(state ? Disabling.enable : Disabling.disable);
```
You don't need the `=== true` here.

Reply:
Is it ok to use falsy/truthy check for boolean value?

Reply:
Yes, as we aren't using coercion here. Remember it's the coercion that we don't like in the code base, so if it's a boolean then we aren't trying to convert it to another type as you would with say `undefined` or a string.